ui/lib/core/addon/components/bar-chart.js

```diff
@@ -0,0 +1,327 @@
+/**
+ * @module BarChart
+ * BarChart components are used to display data in the form of a stacked bar chart, with accompanying legend and tooltip. Anything passed into the block will display in the top right of the chart header.
+ *
+ * @example
+ * ```js
+ * <BarChartComponent @title="Top 10 Namespaces" @description="Each namespace's client count includes clients in child namespaces." @labelKey="namespace_path" @dataset={{this.testData}} @mapLegend={{ array (hash key="non_entity_tokens" label="Active direct tokens") (hash key="distinct_entities" label="Unique Entities") }} @onClick={{this.onClick}} >
```

Comment:
All on one line because storybook didn't like when each arg was on it's own line...